### DISCUSSION TITLE
Does Avatar component not allow non-circular container shapes?

### DISCUSSION THREAD

**Discussion by nikitph** [wix/react-native-ui-lib#2361] [Q&A]
i have been using the following code `<Avatar
size={100}
          name={name}
          containerStyle={{ borderTopLeftRadius: 10 }}
          imageStyle={{ borderRadius: 10 }}
          backgroundColor={generateRowColor(index)}
        />`

but it still renders a circular avatar. is this expected behavior?

**Comment by ethanshar** [comment]
Hi @nikitph 
Good question! Definitely should be checked out, can you please change this to an Issue

**Reply by nikitph** [reply]
Ok will do. Thanks for responding
